### PR TITLE
Enable IP-aliases on kubemark-100 to test

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @wojtek-t

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11122,6 +11122,7 @@
   "ci-kubernetes-kubemark-5-gce": {
     "args": [
       "--cluster=kubemark-5",
+      "--env=KUBE_GCE_ENABLE_IP_ALIASES=true",
```
Comment (wojtek-t) [inline jobs/config.json]: Please put it in .env file.

Comment (wojtek-t) [reply jobs/config.json]: We will forget about it.
In that case, I would even prefer having making a dedicated copy of .env file for that purpose.

Comment (shyamjvs) [reply jobs/config.json]: I changed it to kubemark-100 - PTAL.

**Pull request merged**